openapi.yml

```
@@ -3512,6 +3512,16 @@ paths:
           in: query
           name: created_after
           description: Restricts results to only those that were created after the provided date
+        - schema:
+            type: boolean
+          in: query
+          name: needs_confirmation
+          description: "Restricts results to only those that do not have an associated invite (`needs_confirmation: true`) or do have an associated invite (`needs_confirmation: false`)."
+        - schema:
+            type: integer
+          in: query
+          name: group_visit_id
+          description: Restricts results to only those that are associated to a given group visit (in development).
```

🌵 Not sure what you think about how this reads:
```suggestion
          description: Filters to only those associated with a given `GroupVisit` (in development).
```

This is meant to be an un-released filter? 
Hm... we should double check how we want to show those.

---

This `(in development)` part concerns me...
Is it _just_ this filter that needs to be considered un-released right now?
Or one of the paths even? Or is it the whole model?

I was just talking with Steven about this, trying to figure out the best way to go with those.
I'm thinking that we could conceivably use the tag/label thing to say a path/model is under development and prone to change.
But an attribute like this? We don't really have a way.

Steven was thinking it's better to not have it released to master at all.
If it's needed for contract testing or for the mobile team we can keep branches open and unmerged with the changes in them.

I'm not entirely sure what the long-term solution is yet, but I do prefer not merging in things especially if it's a single attribute rather than a whole path/model we could tag.